spec/controllers/posts_controller_spec.rb

```diff
@@ -1,28 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe PostsController, type: :controller do
+
   describe "GET /new " do
     it "responds with 200" do
-      get :new
+      get :new, params: { get: { session: session[:user_id] = 2 } }
```

Comment:
Why is this user id of 2?

Reply:
it is just a mocked user id to make the test passing

Reply:
This has now changed, these tests are pending